### DISCUSSION TITLE
I want to make it look like it's in this video

### DISCUSSION THREAD

**Discussion by NoteBlockMR** [otomad/OtomadHelper#72] [Q&A] [answered]
https://www.bilibili.com/video/BV1gS4y1G7Jq

Hello, I'm a Korean user

I want to make it like that video, but I have all the textures and sounds

But if you make it, the sound will break, but is there a solution??

https://github.com/otomad/OtomadHelper/assets/170589000/61b5c69d-f381-4f5a-be77-8ae72b5deba1

Sample

**Comment by Evauation** [comment]
So what you are asking for is a way for Vegas Pro to something that can only be done in After Effects, Cinema 4D and Blender. 

Because unfortunately the complexity of the video you are trying to get inspired by can not be done in Vegas Pro. (The first Otomad) utilizes 3d modeling with rhythm keyframing and animated path drawing of blocks across the video.

The audio is breaking because the audio is doing its notes as intended by the MIDI. Setting Legato on with No Looping will fix that issue. But unless you have the cleanest samples to use in Vegas Pro with a right set of effects like Reverb, Delay, Filters, etc. It's not possible to achieve extremely high quality audio mixing with Vegas Pro alone. I'd recommend using FL Studio to do your audio work.

**Comment by NoteBlockMR** [comment]
Where should I set up Legato?

**Comment by Evauation** [comment, marked as answer]
![image](https://github.com/otomad/OtomadHelper/assets/108847709/3a3db947-a8d7-492b-b39e-02052959b535)

These are the settings you would wanna use for using raw samples.
